test/runner.py

```diff
@@ -309,11 +309,7 @@ def load_test_suites(args, modules, start_at, repeat):
 
 
 def flattened_tests(loaded_tests):
-  tests = []
-  for subsuite in loaded_tests:
-    for test in subsuite:
-      tests.append(test)
-  return tests
+  return [test for subsuite in loaded_tests for test in subsuite]
```

Comment:
I think we should avoid nested list comprehensions

Reply:
Reverted but it is good for Python developers to learn how to read comprehensions.